classes/class-log.php

```diff
@@ -138,7 +135,7 @@ function ( $var ) {
 			'blog_id'   => (int) apply_filters( 'wp_stream_blog_id_logged', get_current_blog_id() ),
 			'user_id'   => (int) $user_id,
 			'user_role' => (string) $role,
-			'created'   => (string) $iso_8601_extended_date,
+			'created'   => (string) current_time( 'mysql', true ),
```

Comment:
We always need the UTC timestamp formatted for the [`DATETIME` field](https://dev.mysql.com/doc/refman/8.0/en/datetime.html) in the DB so this will ensure we always get that consistent value.

This is different from the current behaviour of `wp_stream_get_iso_8601_extended_date()` which always adds the timezone information:

https://github.com/xwp/stream/blob/4f85afb87e007c6c968859b821a399f4c9ae2b42/includes/functions.php#L62

which was probably causing the bug as the MySQL client wasn't able to cast it to a valid `DATETIME` value.